packages/volto/src/components/manage/Sidebar/ObjectBrowserNav.jsx

```diff
@@ -135,10 +137,23 @@ const ObjectBrowserNav = ({
                   }
                   trigger={
                     <span>
-                      <Icon
-                        name={getContentIcon(item['@type'], item.is_folderish)}
-                        size="24px"
-                      />
+                      {item['@type'] === 'Image' ||
+                      item.hasPreviewImage === true ||
```

Comment:
```suggestion
                      item.hasPreviewImage ||
```

Just a little redundancy cleanup